rootfs-builder/ubuntu/rootfs_lib.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+check_program(){
+	type "$1" >/dev/null 2>&1
+}
+
+
+
+build_rootfs()
+{
+	# Mandatory
+	local ROOTFS_DIR=$1
+
+	# In case of support EXTRA packages, use it to allow
+	# users add more packages to the base rootfs
+	local EXTRA_PKGS=${EXTRA_PKGS:-""}
+
+
+	check_root
+	mkdir -p "${ROOTFS_DIR}"
+
+	if [ -n "${PKG_MANAGER}" ]; then
+		info "debootstrap path provided by user: ${PKG_MANAGER}"
+	elif check_program "debootstrap" ; then
+		PKG_MANAGER="debootstrap"
+	else
+		die "debootstrap is not installed"
```

Comment:
This block specifies `debootstrap` quite a few times. Could you maybe add a variable to `rootfs-builder/ubuntu/config.sh` and use that variable in all these places?